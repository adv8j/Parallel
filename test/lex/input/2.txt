@TaskGroup g1
{
    int x = 5, y=2;y=3;
    y = 4;
    _task;
    @Task t1
    {
        for (int i = 0; i < 10; i+=1)
        {
            for (int j = 0; j < 10; j+=1);
        }
    }
}
