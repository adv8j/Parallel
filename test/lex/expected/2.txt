TASKGROUP
IDENTIFIER
LBRACE
INT
IDENTIFIER
ASSIGN
INT_LITERAL
COMMA
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
SHARED_IDENTIFIER
SEMICOLON
TASK
IDENTIFIER
LBRACE
FOR
LPAREN
INT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
IDENTIFIER
LT
INT_LITERAL
SEMICOLON
IDENTIFIER
ADD_ASSIGN
INT_LITERAL
RPAREN
LBRACE
FOR
LPAREN
INT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
IDENTIFIER
LT
INT_LITERAL
SEMICOLON
IDENTIFIER
ADD_ASSIGN
INT_LITERAL
RPAREN
SEMICOLON
RBRACE
RBRACE
RBRACE
